manager.go

```diff
@@ -212,6 +212,14 @@ type Options struct {
 	// your program uses "manager.CloneProgram", you might want to enable "KeepKernelBTF". As a workaround, you can also
 	// try to strip as much as possible the content of "KernelTypes" to reduce the memory overhead.
 	KeepKernelBTF bool
+
+	// SkipPerfMapReaderStartup - Perf maps whose name is set to true with this option will not have their reader gorountine started when calling the manager.Start() function.
+	// PerfMap.Start() can then be used to start reading events from the corresponding PerfMap.
+	SkipPerfMapReaderStartup map[string]bool
```

Comment:
Should this be `map[string]struct{}`? Would you ever put a `false` value in this map?

Reply:
It seems to be correct to me to use a boolean as the map will return false whether there is an entry set to false or is there is no entry which would mean "do not skip". Do you have concerns about using a boolean ?